client-haskell/src/Icepeak/Client.hs

```diff
@@ -43,7 +43,6 @@ import Control.Retry (RetryPolicyM, recovering)
 import Data.Aeson (ToJSON)
 import Data.ByteString (ByteString)
 import Data.Foldable (toList)
-import Data.Semigroup ((<>))
```

Comment:
This means that we're probably no longer compatible with old `base` packages where `(<>)` was not exported by the prelude. This either needs conditional compilation, or the version bound needs to be tightened. Or we accept and silence the warning.

Reply:
Good point. We can just accept the warning.